Dockerfile.dev

```diff
@@ -0,0 +1,9 @@
+FROM golang
+RUN apt-get update
+RUN apt-get install -y curl
+RUN curl -sL https://deb.nodesource.com/setup_10.x | bash -
+RUN curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add -
+RUN echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list
+RUN apt-get update
```

Comment:
Do you need to run `apt-get update twice?

Reply:
yes. We're updating the repositories, hence we need to update.